index.js

```diff
@@ -0,0 +1,57 @@
+import Book from './modules/books.js';
+import displayBooks from './modules/display.js';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Thanks for the suggestion, it's because the folder of the module isn't in the repository that it shows that issue, in my local computer I don't have the linters issue.